spa/src/components/account/Profile/InfoTooltip.js

```diff
@@ -0,0 +1,35 @@
+import { ButtonBase, ClickAwayListener } from '@material-ui/core';
+import { Tooltip } from 'components/base';
+import { useState } from 'react';
+import * as S from './InfoTooltip.styled';
```

Comment:
I know that we have the custom to import all styles as `S`, but I think it's really bad to read `<S.Component>` in the JSX. What would you think if importing the components by name? I've been doing that in my PRs and I think it makes them more readable.

```suggestion
import { Icon } from './InfoTooltip.styled';
```

Reply:
I don't have strong feelings about it one way or the other, so let's go with what you're suggesting. Could you add a note on our JS style guide Confluence doc about it?

Updated the files this PR touches in 66e85671e976a5e8dfc9be64ca3b05fad19872a1